hphp/util/asm-x64.h

```diff
@@ -1205,6 +1205,11 @@ struct X64Assembler {
     bytes(n, nops[n]);
   }
 
+  void emitExceptions(int n) {
+    while (n >= 2) ud2();
```

Comment:
So, this looks suspiciously like an infinite loop to me..... (unless `ud2()` is modifying `n` somehow?)